src/Flowdock/FlowdockHandler.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MonologHttp\Flowdock;
+
+use Monolog\Logger;
+use MonologHttp\AbstractHttpClientHandler;
+use Psr\Http\Client\ClientInterface;
+use Psr\Http\Message\RequestFactoryInterface;
+use Psr\Http\Message\RequestInterface;
+
+final class FlowdockHandler extends AbstractHttpClientHandler
+{
+    private const FLOWDOCK_MESSAGES_ENDPOINT = 'https://api.flowdock.com/messages';
```

Comment:
We need to be careful with having the endpoints as as constants.

1. Not sure if flowdock can have a selfhosted solution.
2. if they ever add we will avoid having BC

also I am not quite sure if testing can be more difficult this way.

Reply:
They don't provide a self hosted solution for now but valid point. I need to modify it accordingly. 

What I am thinking is how to match the endpoint with formatters. There are different endpoints for example `Message` and `Private Message` and they accept different payload

Reply:
I would suggest sto create a `private` formatter and a `public` formatter

Then let the developer choose on his own how to initialize the flowdock.. just add a default formatter maybe public one.

Similarly to slack handler/formatters